### DISCUSSION TITLE
I didn't get my student benefits

### DISCUSSION THREAD

**Discussion by whoave** [community/community#119988] [GitHub Education]
### Hi everyone,

Hello, as I've researched it should be activated right away or in 5 days but its been 9 days but still didn't get them or didn't received a mail, what should I do now?
<img width="293" alt="image" src="https://github.com/community/community/assets/39208904/29240b25-fd2e-4e89-9669-3afe8846ea1e">

**Comment by whoave** [comment]
can anyone help me?

**Comment by samus-aran** [comment]
Hi @whoave 👋 

We can see that you have received the Pro upgrade as part of the academic benefits. Can you explain to us what benefits you believe you can not redeem?
